pom.xml

```diff
@@ -71,6 +71,11 @@
             <artifactId>commons-lang</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.jsuereth</groupId>
+            <artifactId>scala-arm_2.11</artifactId>
+            <version>2.0.0-M1</version>
```

Comment:
Friendly reminder that, if we're gonna use this, we have to include this in the parent pom!